src/DssSpell.sol

```diff
@@ -16,135 +16,461 @@
-                240 * WAD,
-                APR_01_2021,
-                3 * 365 days,
-                365 days,
-                SF_001_WALLET
-            )
-        );
+        // Decrease the ETH-A Stability Fee from 2.5% to 2.25%.
+        DssExecLib.setIlkStabilityFee("ETH-A", TWO_PT_TWO_FIVE_PCT_RATE, true);
+
+        // Decrease the ETH-B Stability Fee from 6.5% to 4%.
+        DssExecLib.setIlkStabilityFee("ETH-B", FOUR_PCT_RATE, true);
+
+        // Decrease the WSTETH-A Stability Fee from 3% to 2.5%.
+        DssExecLib.setIlkStabilityFee("WSTETH-A", TWO_PT_FIVE_PCT_RATE, true);
+
+        // Decrease the WBTC-A Stability Fee from 4% to 3.75%.
+        DssExecLib.setIlkStabilityFee("WBTC-A", THREE_PT_SEVEN_FIVE_PCT_RATE, true);
+
+        // Decrease the WBTC-B Stability Fee from 7% to 5%.
+        DssExecLib.setIlkStabilityFee("WBTC-B", FIVE_PCT_RATE, true);
+
+        // Decrease the WBTC-C Stability Fee from 1.5% to 0.75%.
+        DssExecLib.setIlkStabilityFee("WBTC-C", ZERO_PT_SEVEN_FIVE_PCT_RATE, true);
+
+        // Decrease the UNIV2DAIETH-A Stability Fee from 2% to 1%.
+        DssExecLib.setIlkStabilityFee("UNIV2DAIETH-A", ONE_PCT_RATE, true);
+
+        // Decrease the UNIV2WBTCETH-A Stability Fee from 3% to 2%.
+        DssExecLib.setIlkStabilityFee("UNIV2WBTCETH-A", TWO_PCT_RATE, true);
+
+        // Decrease the UNIV2USDCETH-A Stability Fee from 2.5% to 1.5%.
+        DssExecLib.setIlkStabilityFee("UNIV2USDCETH-A", ONE_PT_FIVE_PCT_RATE, true);
+
+        // Decrease the GUNIV3DAIUSDC2-A Stability Fee from 0.5% to 0.25%.
+        DssExecLib.setIlkStabilityFee("GUNIV3DAIUSDC2-A", ZERO_PT_TWO_FIVE_PCT_RATE, true);
+
+        // Decrease the TUSD-A Stability Fee from 1% to 0%.
+        DssExecLib.setIlkStabilityFee("TUSD-A", ZERO_PCT_RATE, true);
+
+
+        /// DIRECT-AAVEV2-DAI (Aave D3M) Target Borrow Rate Decrease
+
+        // Decrease the DIRECT-AAVEV2-DAI Target Borrow Rate from 3.75% to 3.5%.
+        DssExecLib.setD3MTargetInterestRate(DssExecLib.getChangelogAddress("MCD_JOIN_DIRECT_AAVEV2_DAI"), 350); // 3.5%
+
+        /// Maximum Debt Ceiling Changes + GUNIV3DAIUSDC2-A Target Available Debt Increase
+
+        // Decrease the GUNIV3DAIUSDC1-A Maximum Debt Ceiling from 500 million DAI to 100 million DAI.
+        DssExecLib.setIlkAutoLineDebtCeiling("GUNIV3DAIUSDC1-A", 100 * MILLION);
+
+        // Increase the GUNIV3DAIUSDC2-A Maximum Debt Ceiling from 500 million DAI to 750 million DAI.
+        // Increase the GUNIV3DAIUSDC2-A Target Available Debt (gap) from 10 million DAI to 50 million DAI.
+        DssExecLib.setIlkAutoLineParameters("GUNIV3DAIUSDC2-A", 750 * MILLION, 50 * MILLION, 8 hours);
+
+
+        //////////////////////////////////////////////////////////
+        // Set the ESM threshold to 100k MKR
+        // https://vote.makerdao.com/polling/QmQSVmrh?network=mainnet#poll-detail
+
+        require(ESMAbstract(NEW_MCD_ESM).min() == 100_000 * WAD, "DssSpellAction/error-esm-min");
```

Comment:
Can save some bytecode by removing the string error messages here. Just put them in a comment behind it.